src/main/java/bhs/devilbotz/commands/arm/ArmDown.java

```diff
@@ -15,15 +16,19 @@
  */
 public class ArmDown extends CommandBase {
   private final Arm arm;
+  private final Gripper gripper;
 
   /**
    * The constructor for the arm down command.
    *
    * @param arm The arm subsystem.
```

Comment:
add javadoc param for gripper

Reply:
Sorry, I don't have time to add comments to everything.  It's pretty obvious what this is and I don't think it needs a comment.  Arm doesn't have one above either.